src/components/Pages/DefaultPage/DefaultPage.tsx

```diff
@@ -1,7 +1,7 @@
 import { useEffect } from 'react'
 import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
 import { useNavigateWithSearchParams } from '../../../hooks/navigation'
-import { getCurrentConnectionData } from '../../../shared/connection'
+import { getCurrentConnectionData } from '../../../shared/connection/connection'
```

Comment:
WDYT about changing the file name or exporting this in a index?

Reply:
Done!